src/main/java/org/broadinstitute/consent/http/db/DataAccessRequestDAO.java

```diff
@@ -37,9 +38,10 @@ public interface DataAccessRequestDAO extends Transactional<DataAccessRequestDAO
   @UseRowReducer(DataAccessRequestReducer.class)
   @SqlQuery(
       """
-          SELECT dd.dataset_id, dar.id, dar.reference_id, dar.collection_id, dar.parent_id, dar.user_id, dar.create_date, dar.sort_date, dar.submission_date, dar.update_date,
+          SELECT collection.dar_code, dd.dataset_id, dar.id, dar.reference_id, dar.collection_id, dar.parent_id, dar.user_id, dar.create_date, dar.sort_date, dar.submission_date, dar.update_date,
             (regexp_replace(dar.data #>> '{}', '\\\\u0000', '', 'g'))::jsonb AS data FROM data_access_request dar
           LEFT JOIN dar_dataset dd on dd.reference_id = dar.reference_id
+          LEFT JOIN dar_collection collection on collection.collection_id = dar.collection_id
```

Comment:
Joining to enrich with dar_code

Reply:
Has the performance of any of these queries changed?

Reply:
Not in a noticeable way at this point.  That being said, I think there are a number of queries that we may have to keep an eye on as we start using dar_collection in new ways.  That may cause is to want to consider moving dar_code into data_access_request and simplifying the relationships we have going on in all of this code.